src/navigation/Router.tsx

```diff
@@ -32,7 +34,8 @@ const router: React.FC<RouterProps> = props => (
 );
 
 const mapStateToProps = (state: AppState) => ({
-  isLoggedIn: state.isLoggedIn,
+  auth: state.authReducer.auth,
```

Comment:
not clear value from reducer
I mean 'auth' this is process, isAuth/isLoggedIn much more clear var name for that